app/views/admin/dashboard/index.html.erb

```diff
@@ -0,0 +1,27 @@
+<% @orders.each do |order| %>
```

Comment:
This looks great. Will we want to go in and create sections for each order as well?